sale_order_blanket_order/__manifest__.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2024 ACSONE SA/NV
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+{
+    "name": "Sale Order Blanket Order",
+    "summary": """Manage blanket order and call of ordr""",
+    "version": "16.0.1.0.0",
+    "license": "AGPL-3",
+    "author": "ACSONE SA/NV,BCIM,Odoo Community Association (OCA)",
+    "website": "https://github.com/OCA/sale-workflow",
+    "depends": [
+        "sale_manual_delivery",
```

Comment:
@lmignon The install of this addon does not work on large database.

See: https://github.com/OCA/sale-workflow/issues/3476

Reply:
@rousseldenis 
It worked on my very large database.... (even if it was not fast and should be improved)